mergify_engine/branch_updater.py

```diff
@@ -24,8 +24,17 @@
 
 LOG = daiquiri.getLogger(__name__)
 
+AUTHENTICATION_FAILURE_MESSAGES = [
```

Comment:
nit: `set()`:p

Reply:
or even a tuple ;)

Reply:
Well a `tuple` is ordered, whereas a `set` is not. In that case we don't care about order, a `set` makes that clear. ;)